xml/System.ServiceModel.Syndication/XmlUriData.xml

```diff
@@ -110,9 +115,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the UriString property.</summary>
```

Comment:
The `UriString` should be formatted as code.

Reply:
@BillWagner do you mean a full `<see cref="..." />` or do you mean `<c>...</c>`?